app/models/webhook_delivery.rb

```diff
@@ -4,6 +4,8 @@ class WebhookDelivery < ApplicationRecord
   enum event: {
          submission_created: 'submission.created',
          submission_graded: 'submission.graded',
+         submission_verified: 'submission.verified',
+         submission_automatically_verified: 'submission.automatically.verified',
```

Comment:
Shouldn't this be `submission.automatically_verified`? We should probably stick to a `resource.event` naming scheme for non-Rails consumers (with the _period_ appearing only once).

```suggestion
         submission_automatically_verified: 'submission.automatically_verified',
```